app/forms/declaration/RoutingQuestionYesNo.scala

```diff
@@ -33,10 +33,11 @@ object RoutingQuestionYesNo {
 
   val allowedValues: Seq[String] = Seq(yes, no)
 
-  def form(): Form[Boolean] = Form(
+  def form(cachedCountries: Seq[String] = Seq.empty): Form[Boolean] = Form(
     Forms.mapping(
       "answer" -> requiredRadio("declaration.routingQuestion.empty")
         .verifying("declaration.routingQuestion.error", isContainedIn(allowedValues))
+        .verifying(s"declaration.routingCountries.limit", _ => cachedCountries.length < DestinationCountries.limit)
```

Comment:
if you reach 99 containers you won't be able to submit "No" version

Reply:
Fixed, please check now